schemas/input/csl-data.json

```diff
@@ -20,20 +20,22 @@
         {
           "full": "Finis Coronat Opus: A Curious Reciprocity; Shelley’s “When the Lamp Is Shattered”",
           "main": "Finis Coronat Opus",
```

Comment:
Where have settled on regarding punctuation? Shouldn't we include the colon in "main"?
`"main": "Finis Coronat Opus: "`
(Of course only if the colon appears in the original...)

Reply:
Keeping with the "let's keep simple things simple" approach, I think there should be a convention to add a default delimiter if one doesn't exist, so that adding colons and such (by FAR the most common convention) should be unnecessary.

Reply:
Ok. That might mean we will want to add an option `normalize-delimiters="false"`. Then you could still add `title-delimiter=": "` to a style, which will then add a colon if no punctuation is present, but it will not override an existing period. (Exclamation marks and question marks will be protected anyway.)

Reply:
Should title delimiters also be defined in locales?

Like, in English, would be a colon.

Reply:
Better not do this. If I use Chicago, I'll always want to have colons as title-subtitle-delimiter, even when writing in German. But a institutional style from a German university will perhaps use periods.

Reply:
The point is, in English at least, I've literally never seen anything other than colons. 

So do we require the fallback to be specified in a style, or allow it to be in a locale? The style can override it after all.

So in your German case, maybe the period is in the locale, but the colon is in Chicago?

Reply:
We can allow it in a locale, but in one locale only so that there really is one standard delimiter (bit one per locale). Styles that require a different delimiter can still override this.

Reply:
In that case, we can just define colon as the default delimeter.

Reply:
@gduffner What do you think about this? I remember you've had [doubts](https://discourse.citationstyles.org/t/handling-main-sub-title-splits-citeproc-js/1563/12?u=denis_maier) regarding localizing punctuation...

Reply:
It seems his concern was about user experience; unexpected behavior.

I'd agree that should be an important consideration, but that this should be a part of it.

Reply:
> @georgd What do you think about this? I remember you've had [doubts](https://discourse.citationstyles.org/t/handling-main-sub-title-splits-citeproc-js/1563/12?u=denis_maier) regarding localizing punctuation...

My impression is that the delimiter is a locale-independent property of the style – at least I’ve never come across anything else. A style that requires colon in English will be used with colon in a German document as well, as @denismaier said. Localisation for special typographical conventions (like the space before colon in French) is not restricted to this case, likewise script specific localisation (e.g. ano teleia instead of colon in Greek script). That’s why I thought that a simple global property would be enough.

But what do you expect to do with non-latin titles? I suppose they’re currently not normalised?